Gemfile

```diff
@@ -28,6 +28,7 @@ gem 'gettext_i18n_rails', '~> 1.0'
 gem 'i18n', '~> 0.6.4'
 gem 'turbolinks', '~> 2.5'
 gem 'logging', '>= 1.8.0', '< 3.0.0'
+gem 'strong_parameters'
```

Comment:
This should probably be pinned a little, but I'm not sure whether to `0.2.x` or `~> 0.2`?